config/webpacker.yml

```diff
@@ -87,7 +87,7 @@ production:
   <<: *default
 
   # Production depends on precompilation of packs prior to booting for performance.
-  compile: false
+  compile: true
```

Comment:
Can you say more about why this is necessary? It's a pretty significant change to our production settings so would like to avoid it if possible.

Reply:
Hmm. I think we unintentionally did this somehow so I reverted it and pushed it back up

Reply:
Ah, that makes sense. In that case, i think this is ready. Thank you!

Reply:
Thank you @exbinary! Is there anything else we need to do or will someone with write-access merge?

Reply:
Nope, merging now. Thanks again!